packages/nx/src/tasks-runner/pseudo-terminal.ts

```diff
@@ -5,6 +5,7 @@ import { ChildProcess, IS_WASM, RustPseudoTerminal } from '../native';
 import { PseudoIPCServer } from './pseudo-ipc';
 import { RunningTask } from './running-tasks/running-task';
 import { codeToSignal } from '../utils/exit-codes';
+import { NormalizedRunCommandsOptions } from '../executors/run-commands/run-commands.impl';
```

Comment:
This introduces a circular dep. `run-commands.impl` already imports from this file. We can either extract the type of a shared file or duplicate it here.